src/AddFoodForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useState } from 'react';
+import { Divider, Input } from 'antd';
+
+
+function AddFoodForm({newForm}) {
+    const [name, setName] = useState('');
+    const [image, setImage] = useState('');
+    const [calories, setCalories] = useState();
```

Comment:
Here,
```
const [calories, setCalories] = useState();
const [servings, setServings] = useState();
```

it would've been better to use 0 since it's a number, like:
```
const [calories, setCalories] = useState(0);
const [servings, setServings] = useState(0);
```